effekt/shared/src/main/scala/effekt/Compiler.scala

```diff
@@ -115,6 +115,11 @@ trait Compiler[Executable] {
    * - Server / Driver to typecheck and report type errors in VSCode
    */
   def runFrontend(source: Source)(using C: Context): Option[Module] =
+    def getStackTrace(e: Throwable): String =
+      val stringWriter = new java.io.StringWriter()
+      e.printStackTrace(new java.io.PrintWriter(stringWriter))
+      stringWriter.toString
```

Comment:
This part is essentially cherry-picked from the discussion in @JakubSchwenkbeck's #731. Thanks!